spec/fixtures/prover_outputs/CounterSatisfiable/Vampire

```diff
@@ -0,0 +1,29 @@
+Calculus      : VampireResCalc
+Spent time    : CPU time        = 00:00:00.295999999999
+                Wall clock time = 00:00:00.294999999999
+Prover output :
+Satisfiability detected
```

Comment:
Oh, I named the corresponding ontology `CounterSatisfiable` (see re-written Simple_Implications.casl) because of what the other provers report as SZS status.